.github/workflows/cicd-main.yml

```diff
@@ -4290,6 +4290,104 @@ jobs:
         - uses: "NVIDIA/NeMo/.github/actions/cancel-workflow@main"
           if: "failure()"
 
+  L2_Megatron_GPT_Pretraining_and_Resume_Training_Torch_Dist_TP2_PP2:
```

Comment:
Instead of adding a new test can we update the existing tests to use torch dist?